src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/Delegation/SnippetResponseRewriterTest.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.LanguageServer.Completion.Delegation;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Test.Completion.Delegation;
+public class SnippetResponseRewriterTest(ITestOutputHelper testOutput)
```

Comment:
nit: please add blank a line between the namespace and class declarations.

Reply:
About "..." - same as above, will do whatever we decide, though in general no other snippets in any other language use "..." and instead rely on snippet glyph and snippet description/tooltip to indicate it's a snippet. Happy to keep "..." if we all like it better.

Reply:
Re-added per sync-up discussion